Makefile

```diff
@@ -2,7 +2,7 @@ ENABLE_WASM_OPT ?= true
 
 .PHONY: build
 build:
-	tinygo build -wasm-abi=generic -target=wasi -gc=leaking -o redirect.wasm redirect.go
```

Comment:
Is `-wasm-abi=generic` actually needed here?

Reply:
This flag has been removed as of tinygo 0.27, so it makes sense to drop.  However, we'd probably want to also bump the tinygo version (and go version?) used in CI accordingly then.  Not sure if it is safe to make this change and then create a follow-up to bump tinygo/go versions, or if they should be done together in its own PR...

Reply:
I built it locally `0.27.0` without any problems. However, I didn't bump the tinygo version because I stumbled upon [this note](https://developer.fermyon.com/spin/go-components#versions) in the spin documentation.

Unfortunately, I wasn't able to spot any further details why or where it would actually break with newer releases of tinygo.